src/lists/PreferencesList.stories.tsx

```diff
@@ -17,13 +17,11 @@ export default {
 
 const listingPreferences = [
   {
-    ordinal: 1,
     links: [],
     title: "Title 1",
     description: "Description 1",
   },
   {
-    ordinal: 2,
```

Comment:
were these supposed to be deleted? [storybook](https://deploy-preview-10--storybook-bloom-dev.netlify.app/?path=/story/lists-preferences-list--preferences) looks a little off imo.